app/scripts/proactive/rest/studio-client.js

```diff
@@ -327,6 +348,7 @@ define(
                         that.alert("Cannot submit the job", result.errorMessage, 'error');
                     } else if (result.id) {
                         that.alert("Job submitted", "<html></html><a href='/scheduler' target='_blank'>'" + result.readableName + "' submitted successfully (Id " + result.id + ")</a></html>", 'success');
+                        that.alert("Job submitted", "<html></html><a href='/automation-dashboard/#/job-info?jobid=" + result.id + "&tab=0' target='_blank'>'" + result.readableName + "' submitted successfully (Id " + result.id + ")</a></html>", 'success');
```

Comment:
I believe we need to have both links inside the same alert message.

Also we should rewrite it in order to understand these two links.
e.g.
`"<label>" + result.readableName + "' submitted successfully (Id " + result.id + ")+ "</label><br>"` 
then append the links (starting with automation dashboard) with an inside text like Open in Workflow Execution, Open in Scheduler